tools/project_manager/projectmgr.js

```diff
@@ -13,7 +13,9 @@ window.addEventListener("DOMContentLoaded", () => {
     });
 
     function showTime() {
-        timeSpan.textContent = dateTimeFormat.format(Date.now());
+        if (timeSpan) {
```

Comment:
minor, but you could put the if before even setting up the interval or anything to save CPU cycles every minute, but yeah...not a big deal.